src/clojure/crap/poop.clj

```diff
@@ -0,0 +1,34 @@
+(ns crap.poop
```

Comment:
Thanks for this one Matt! you are awesome;).
I'd like to keep the old one tho - I'd like the fact that was so minimal. Can we rename this updated one  *number two* ;) ?